client/src/document/index.scss

```diff
@@ -642,6 +642,7 @@ kbd {
   max-height: var(--max-height);
   position: sticky;
   top: 5rem;
+  z-index: var(--z-index-top);
```

Comment:
```suggestion
```
let's just fix the issue at hand:

![image](https://user-images.githubusercontent.com/3604775/231839666-16506955-db1a-47e6-8e69-6ce7a772c370.png)